.golangci.yml

```diff
@@ -0,0 +1,42 @@
+run:
+  timeout: 5m
+  tests: false
+output:
+  formats:
+    - format: github-actions
+    - format: colored-line-number
+linters:
+  disable-all: true
+  enable:
+    - errcheck # all errors checked
+    - gosimple # simplify code
+    - govet # run govet
+    - ineffassign # check ineffectual assignments
+    - staticcheck # staticcheck
+    - typecheck # parse and check go code
+    - unused # unused constants, variables, functions and types
+    - revive # replacement for golint
+    - dupl # check duplicated code
+    - goconst # check strings that can turn into constants
+    - gofmt # check fmt
+    - goheader # Check license headers, only checks files in current year
+    - goimports # check imports
+    - gocyclo # check complexity
+linters-settings:
+  gocyclo:
+    min-complexity: 20 # default is 30 which is too high
+  goheader:
+    template: |-
+      Copyright © 2022 - {{year}} SUSE LLC
```

Comment:
As some code is already taken from Elemental with 2022 as the base year I think it's fine to keep it as is.